trick_source/java/test/Makefile

```diff
@@ -0,0 +1,84 @@
+# To run new unit tests, add new tests to the "test" target
+
+TRICK_HOME := $(abspath $(dir $(lastword $(MAKEFILE_LIST)))../../..)
+
+# Get JAVAC definition
+include ${TRICK_HOME}/share/trick/makefiles/Makefile.common
+
+JAVAC ?= javac
+JAVAC_FLAGS = -g -Xlint:unchecked -Xlint:deprecation
+# delineate 'javac -version' for version number 
+JAVAC_VERSION := $(shell ${JAVAC} -version 2>&1 | perl -ne 'print /(\d+)/')
+
+# include java.se.ee for JAXB xml annotations (for trick-tv) in java version 9
+ifeq ($(JAVAC_VERSION),9)
```

Comment:
This only applies to Java 10 because it is being deprecated in Java 11.  I will add a check for 10 as a short-term solution and create an issue for a long term solution

Reply:
It was deprecated in 9. I believe you mean it will be *removed* in 11?

Reply:
Yes, it will be removed in 11. And ouch, that italics destroyed my ego.

Reply:
You can always go back and modify your comment, making *me* look like the idiot :wink: